### DISCUSSION TITLE
[Feature] QOL jump to time

### DISCUSSION THREAD

**Discussion by 1Euro7Cent** [immich-app/immich#7042] [Feature Request]
### The feature

QOL request: Quality of life.

For example, I know that I was on a party on date 23.4.2021, and I want to get the photos from that day.
Currently, I need to scroll to about that time and then scroll and find it. I would like to insert a date and it auto jumps to it or the nearest to it.

With over 10K assets it is difficult:
![image](https://github.com/immich-app/immich/assets/67194495/cb63ca9a-9176-41fc-8429-650723e4200d)

(don't want to scroll here):
![image](https://github.com/immich-app/immich/assets/67194495/577b2528-d383-4e66-8146-0f9b40bcc775)

For example in VS Code I can jump straight to a line

![image](https://github.com/immich-app/immich/assets/67194495/3a367e2f-f56a-4820-bc93-4405313750a5)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3873
